portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,15 +19,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays; 
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comment_list = new ArrayList<>(Arrays.asList("HE WAS NUMBER 1", 
+  "Wow. Guess what comment this is. Dos!", "Three! THREE! ThIrD yerrr"));;
```

Comment:
It will be cleaner:
```
List<String> list = Arrays.asList("hello", "hi");
```

Reply:
The instructions specify to make it an ArrayList<String>.

Reply:
Interesting. `Arrays.asList` accepts an array of String as arguments already, it returns an array list backed by an array ([see official doc](https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#asList(T...))), I don't see a reason to initiate another `new ArrayList<>` of it. I think the instruction should be updated/improved. lol.